boltdesignsystem/bolt
# test: Replace unnecessary e2e test with Jest test for renderable items

## Jira

N/A

## Summary

Replaces unnecessary e2e test with Jest test for renderable items in bolt list

## Details

It turns out I was using an unnecessary e2e test because I just didn't know how flexible `renderString()` was.

## How to test

- Revert the changes in this commit: 701726ac8deaa7f09c1ce6a2badaaa8dea1c1fc9
- Confirm that the test fails
- Confirm that the test passes after those changes are introduced.
